app/views/newflow/login_signup/profile_newflow.html.erb

```diff
@@ -0,0 +1,175 @@
+  <div class="row">
+    <div class="col-sm-2 category">
+      Username
+    </div>
+    <div class="col-sm-10 info">
+      <a href="#" id="username" class="editable" data-name="username" data-type="text"><%= current_user.username %></a>
+    </div>
+  </div>
+  <% end %>
+
+  <div class="row">
+    <div class="col-sm-2 category">
+      Email addresses
+    </div>
+    <div id="email-template" style="display:none">
+      <%= email_entry(value: "", id: "", is_verified: false, is_searchable: "") %>
+    </div>
+    <div class="col-sm-10 info">
+      <% current_user.email_addresses.order(:created_at).each do |email| %>
+        <%= email_entry(value: email.value, id: email.id, is_verified: email.verified, is_searchable: email.is_searchable) %>
+      <% end %>
+      <%= link_to 'Add email address', '#', {id: 'add-an-email'} %>
+    </div>
+  </div>
+
+  <%
+    # all_providers = %w(facebook google_oauth2 identity twitter)
+    all_providers = OmniauthData::VALID_PROVIDERS
+
+    # The `all providers &` enforces a standard sort order
+    current_providers = all_providers & current_user.authentications.collect(&:provider)
+    other_providers = all_providers - current_providers - ['twitter', 'google_oauth2', 'facebook']
+  %>
+
+  <div class="row enabled-providers">
+    <div class="col-sm-2 category">
+      <%= 'My login methods' %>
+    </div>
+    <div class="col-sm-10 info providers">
+      <% current_providers.each do |provider| %>
+        <%= newflow_way_to_login(provider: provider, has_authentication: true, current_providers: current_providers) %>
+      <% end %>
+    </div>
+    <% if other_providers.any? %>
+    <div class="col-sm-offset-2 info col-sm-10"><%= link_to (I18n.t("student_profile.enable_other_sign_in_options")), nil, id: 'enable-other-sign-in' %></div>
+    <% end %>
+
+  </div>
+
+  <div class="row other-sign-in">
+    <div class="col-sm-2 category">
+      <%= t("student_profile.other_sign_in_options_html") %>
+    </div>
+    <div class="col-sm-10 info providers">
+      <% other_providers.each do |provider| %>
+        <%= newflow_way_to_login(provider: provider, has_authentication: false, current_providers: current_providers) %>
+      <% end %>
+    </div>
+  </div>
+
+<% end %>
+
+<%= render partial: 'layouts/newflow_footer' %>
+
+<%= content_for :javascript do %>
+<script type="text/javascript">
+
+$.fn.editable.defaults.mode = 'inline';
+$.fn.editable.defaults.send = "always";
+$.fn.editable.defaults.ajaxOptions = {type: "PUT"};
+$.fn.editable.defaults.url = "<%= profile_path %>";
+
+$('#username').editable({
+  success: function(response, newValue) {
+    $('#session-info .username').text(newValue);
+  }
+});
+
+OX.Profile.Name.editable($('#name'), {
+    title: "<%= current_user.title %>",
+    first_name: "<%= current_user.first_name %>",
+    last_name: "<%= current_user.last_name %>",
+    suffix: "<%= current_user.suffix %>"
+});
+
+
+</script>
+<% end # javscript content block %>
```

Comment:
There's quite a few bare strings that should be translated here.   My Account , Username, Add email address etc.